ui/app/components/job-status/allocation-status-block.hbs

```diff
@@ -5,13 +5,39 @@
   {{#if this.countToShow}}
     <div class="ungrouped-allocs">
       {{#each (range 0 this.countToShow)}}
-        <span class="represented-allocation {{@status}}" />
+        <span class="represented-allocation {{@status}} {{@health}} {{@canary}}">
+          {{#unless @steady}}
+            {{#if (eq @status "running")}}
+              <span class="alloc-health-indicator">
+                {{#if (eq @health "healthy")}}
+                  <FlightIcon @name="check" @color="white" />
+                {{else}}
+                  <FlightIcon @name="running" @color="white" />
+                {{/if}}
+              </span>
+            {{/if}}
+          {{/unless}}
+        </span>
       {{/each}}
     </div>
   {{/if}}
   {{#if this.remaining}}
-    <span class="represented-allocation rest {{@status}}">
+    <span class="represented-allocation rest {{@status}} {{@health}} {{@canary}}">
       {{#if this.countToShow}}+{{/if}}{{this.remaining}}
+      {{#unless @steady}}
+        {{#if (eq @canary "canary")}}
```

Comment:
This one I'm less inclined to combine; this second condition (canary check) is only for a smaller part of the larger unless.

In the example you suggested earlier it makes sense as both conditions wrap the same amount of template. Here, though, it would only result in more LoC.